### ISSUE TITLE
Set maximum video length

### ISSUE THREAD

**Issue by ibhavin** [dev-labs-bg/swift-video-generator#5]
How can I set maximum duration of video for single video and single audio merge?

**Comment by tgeorgieva** [comment]
Use the `duration` parameter in the `setup(withImages)` function.

**Comment by ibhavin** [comment]
@tgeorgieva  I have give static value to duration like this:
```
//original
duration = max(_duration, Double(CMTime(seconds: minSingleVideoDuration, preferredTimescale: 1).seconds)) 
// static value
duration = 5
```

but still video generated by audio length.

**Comment by tgeorgieva** [comment]
In the `mergeAudio(withVideoURL)` there is another `duration` parameter which gets incremented with the audio length when it's being added to the generated video. 

Keep in mind, setting the max length of a video like that doesn't mean the audio will get faster or the frame duration shorter, just means it'll cut off at the time the video stops. You can try doing that, changing the frame time but the audio manipulation will be tricky.

**Comment by ibhavin** [comment]
@tgeorgieva 
I have also tried to set duration in `mergeAudio(withVideoURL)`:

```
duration = duration + audioAsset.duration
duration = CMTime(seconds: Double(5), preferredTimescale: 1)
```

but still video length is equal to audio length.

**Comment by tgeorgieva** [comment]
@ibhavin This has been implemented in version 1.0.7. It's a property that you can set to single and multiple video generation.
**maxVideoLengthInSeconds**
